spec/requests/forecasting/distributions_request_spec.rb

```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
+RSpec.describe "Forecasting::Distributions", type: :request do
+
+end
```

Comment:
This can *probably* be deleted since you've only got the one controller action and there isn't any special logic in the request process. 

If you wanted to do a single spec for "#index is successful" or something that's fine too